st2api/st2api/controllers/v1/actionalias.py

```diff
@@ -156,13 +156,16 @@ def post(self, action_alias):
 
         return action_alias_api
 
-    @request_user_has_resource_db_permission(permission_type=PermissionType.ACTION_MODIFY)
+    @request_user_has_resource_db_permission(permission_type=PermissionType.ACTION_ALIAS_MODIFY)
     @jsexpose(arg_types=[str], body_cls=ActionAliasAPI)
-    def put(self, action_alias_ref_or_id, action_alias):
+    def put(self, action_alias, action_alias_ref_or_id):
```

Comment:
Yeah, I was personally not a fan of that change and I prefer id to be first (this way it's consistent with jsexpose and everything and less magic, etc.), but @enykeev some how managed to continence me in https://github.com/StackStorm/st2/pull/2841 that this is good idea :P